less/matching.less

```diff
@@ -1,240 +1,111 @@
+            right: inherit;
+            left: 10px;
+        }
+
+        .matching-select-icon {
+            color: @item-text-color;
+        }
+
+        .matching-correct-icon {
+            color: @validation-success;
+            display: none;
+        }
+
+        .matching-incorrect-icon {
+            color: @validation-error;
+            display: none;
+        }
+
+    }
+
+    .matching-widget {
+
+        &.submitted {
+            .matching-select {
+                cursor: default
+            }
+            .matching-select-container {
+                background-color: @item-color-disabled;
+            }
+
+            .matching-select-state {
+                z-index: 1;
+            }
+        }
+
+        &.show-user-answer {
+
+            .matching-select-icon {
+                display: none;
+            }
+            .incorrect {
+                .matching-incorrect-icon {
+                    display: block;
+                }
+            }
+            .correct {
+                .matching-correct-icon {
+                    display: block;
+                }
+            }
+        }
+
+        &.show-correct-answer {
+
+            .matching-select-icon {
+                display: none;
+            }
+            .incorrect .matching-correct-icon,
+            .correct .matching-correct-icon {
+                display: block;
+            }
+        }
+
+    }
 
 }
 
 .no-touch {
-	.matching-select-icon {
-		&:hover {
-			color: @item-text-color-hover;
-		}
-	}
-	.matching-select-container {
-		&:hover {
-			color: @button-text-color-hover;
-			background-color: @item-color-hover;
-		}
-	}
-}
-
-.ie8 {
-	.matching-select-container {
-		&.disabled {
-			background-color: @item-color-disabled;
-		}
-	}
-}
-
-.select2-container--default,
-.dir-rtl .select2-container--default {
-	ul, li {
-		margin: 0;
-	}
-	
-	span.select2-dropdown {
-		background-color: @item-color;
-		color: @item-text-color;
-	}
-
-	.select2-results__option {
-		padding-top: (@item-padding-top / 2);
-		padding-bottom: (@item-padding-bottom / 2);
-		padding-left: @item-padding-left;
-		padding-right: @item-padding-right;
-	}
-
-	li.select2-results__option--highlighted[aria-selected],
-	li.select2-results__option[aria-selected=true] {
-		background-color: @item-color-hover;
-		color: @item-text-hover-color;
-	}
-
-	&.select2-container--disabled {
-
-		span.select2-selection--single,
-		.no-touch & span.select2-selection--single:hover {
-			background-color: @item-color-disabled;
-			color: @item-text-disabled-color;
-		}
-	
-	}
-}
-
-.accessibility.no-touch .matching-component .matching-widget.disabled .select2-container.select2-container--focus,
-.accessibility.no-touch .matching-widget.disabled .select2.select2-container *:focus {
-	outline: 0;
-}
-
-// fixes IE8 scrolling jump issue when item focused 
-.select2-search__field {
-	position: fixed;
+    .matching-select-icon {
+        &:hover {
+            color: @item-text-color-hover;
```

Comment:
Ticks and crosses go white on hover.

Reply:
this comes out of the theme now, have fixed it in the component

Reply:
Do we need an issue creating for this?

Reply:
https://github.com/adaptlearning/adapt_framework/issues/2366